src/core/lib/security/credentials/tls/tls_utils.h

```diff
@@ -46,6 +46,20 @@ absl::string_view GetAuthPropertyValue(grpc_auth_context* context,
 std::vector<absl::string_view> GetAuthPropertyArray(grpc_auth_context* context,
                                                     const char* property_name);
 
+struct EVP_PKEYDeleter {
+  void operator()(EVP_PKEY* pkey) const { EVP_PKEY_free(pkey); }
```

Comment:
You'll need to include openssl/evp.h, openssl/bio.h, and openssl/x509.h in order to reference the various free functions.

Reply:
Thanks, noted.